### DISCUSSION TITLE
Use of capacitor to develop a HIPAA-compliant app

### DISCUSSION THREAD

**Discussion by b2zv** [ionic-team/capacitor#4971] [Q&A]
I'd like to use capacitor with a nextjs app to read HealthKit data from the user's iPhone. Are there any steps that I need to take with Capacitor's configuration or at runtime in order to be compliant with HIPAA?

**Comment by mlynch** [comment]
Hey @b2zv. Take a look at this post for some common considerations when building HIPAA-compliant mobile apps: https://ionicframework.com/blog/hipaa-compliant-mobile-apps-with-ionic/

You’re responsible for ensuring PII is protected and following the tips in that post will help you do that. This is outside of the scope of the OSS Capacitor project.

**Reply by b2zv** [reply]
Hi mlynch - thanks for the quick response. I will take a look at the link. I wasn't sure if capacitor stores any data in-memory/on-disk that should be encrypted while it bridges with native APIs. Hence the question.
